iaaip/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template "><head>
+  <meta charset="utf-8">
+  <title>
+    Start your appeal - Appeal an immigration or asylum decision - HMCTS
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+  <meta name="theme-color" content="#0b0c0c">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+
+  <link rel="shortcut icon" href="/assets/images/favicon.ico" type="image/x-icon">
+
+
+
+  <base href="https://localhost:3000">
```

Comment:
Get rid of this

Reply:
this is what's causing the rendering to be broken for me